src/test/jtx/impl/WSClient.cpp

```diff
@@ -208,7 +208,116 @@ class WSClientImpl : public WSClient
     Json::Value
     invoke(Json::Value const& cmd) override
     {
-        return {};
+        using boost::asio::buffer;
+        using namespace std::chrono_literals;
+        Json::Value request;
+        if (cmd.isArray())
+        {
+            auto const num_requests = cmd.size();
+            for (unsigned i = 0; i < num_requests; ++i)
+            {
+                Json::Value const& cmdi = cmd[i];
+
+                if (cmdi.isMember(jss::params))
+                {
+                    auto const& params = cmdi[jss::params][0u];
```

Comment:
Is this where that `extract_params` function could help? 

It might be nice to have; there is a lot similar code in this function and I'd find it easier to read with less repetition.